examples/03_connectivity/plot_seed_based_correlation.py

```diff
@@ -0,0 +1,176 @@
+brain_masker = input_data.NiftiMasker(
+    smoothing_fwhm=6,
+    detrend=True, standardize=True,
+    low_pass=0.1, high_pass=0.01, t_r=2.,
+    memory='nilearn_cache', memory_level=1, verbose=0)
+
+##########################################################################
+# Then we extract the brain-wide voxel-wise time series while regressing
+# out the confounds as before
+brain_time_series = brain_masker.fit_transform(func_filename,
+                                               confounds=[confound_filename])
+
+
+##########################################################################
+# We can now inspect the extracted time series. Note that the **seed time
+# series** is an array with shape n_volumes, 1), while the
+# **brain time series** is an array with shape (n_volumes, n_voxels).
+
+print("seed time series shape: (%s, %s)" % seed_time_series.shape)
+print("brain time series shape: (%s, %s)" % brain_time_series.shape)
+
+##########################################################################
+# We can plot the **seed time series**.
+
+import matplotlib.pyplot as plt
+
+plt.plot(seed_time_series)
+plt.title('Seed time series (Posterior cingulate cortex)')
+plt.xlabel('Scan number')
+plt.ylabel('Normalized signal')
+plt.tight_layout()
+
+##########################################################################
+# Exemplarily, we can also select 5 random voxels from the **brain-wide
+# data** and plot the time series from.
+
+plt.plot(brain_time_series[:, [10, 45, 100, 5000, 10000]])
+plt.title('Time series from 5 random voxels')
+plt.xlabel('Scan number')
+plt.ylabel('Normalized signal')
+plt.tight_layout()
+
+
+
+##########################################################################
+# Performing the seed-based correlation analysis
+# ----------------------------------------------
+#
+# Now that we have two arrays (**sphere signal**: (n_volumes, 1),
+# **brain-wide voxel-wise signal** (n_volumes, n_voxels)), we can correlate
+# the **seed signal** with the **signal of each voxel**. The dot product of
+# the two arrays will give us this correlation. Note that the signals have
+# been variance-standardized during extraction. To have them standardized to
+# norm unit, we further have to divide the result by the length of the time
+# series.
+import numpy as np
+
+seed_based_correlations = np.dot(brain_time_series.T, seed_time_series) / \
+                          seed_time_series.shape[0]
+
+################################################
+# The resulting array will contain a value representing the correlation
+# values between the signal in the **seed region** of interest and **each
+# voxel's signal**, and will be of shape (n_voxels, 1). The correlation
+# values can potentially range between -1 and 1.
+print("seed-based correlation shape: (%s, %s)" % seed_based_correlations.shape)
+print("seed-based correlation: min = %.3f; max = %.3f" % (
+seed_based_correlations.min(), seed_based_correlations.max()))
```

Comment:
Code formatting guidelines say that you should indent the "seed_based_correlation" to match the right after the opening parenthesis.